util/capacity-reservation.py

```diff
@@ -58,20 +62,57 @@ def parse_args():
     return parser.parse_args()
 
 
-def create_reservation(client, instance_type, zone, count, tag_path, dry_run):
+def _retrieve_tags(client, instance_id):
+    """Retrieve tags from an EC2 instance"""
+    response = client.describe_tags(
+        Filters=[
+            {
+                'Name': 'resource-id',
+                'Values': [instance_id]
+            }
+        ]
+    )
+    if 'Tags' in response:
+        tags = [
+            {'Value': tag['Value'], 'Key': tag['Key']}
+            for tag in response['Tags']
+            if tag['Key'] in ['Department', 'Name', 'Project', 'SubProject']
+            ]
+        return tags
+    else:
+        metadata = response['ResponseMetadata']
```

Comment:
will you ever hit this condition?  seems like describe_tags will either return a list of tags or an empty list.

Reply:
right, I'll do a length check instead